internal/provider/store.go

```diff
@@ -118,6 +119,12 @@ func (s *ProviderStore) SkipTeardown() bool {
 	return v != ""
 }
 
+// PortForwards returns the IMAGETEST_PORT_FORWARDS environment variable.
+func (s *ProviderStore) PortForwards() []string {
+	v := os.Getenv("IMAGETEST_PORT_FORWARDS")
```

Comment:
This has the potential of causing trouble if the empty string isn't handled. Suggestion:
```suggestion
	v, ok := os.LookupEnv("IMAGETEST_PORT_FORWARDS")
	if !ok || "" == v {
	    return nil
	}
```